batch.py

```diff
@@ -461,22 +464,32 @@ def run_gene_association(
 
 
 def summarise_association_results(
-    *pv_dfs: list[str],
+    pv_dfs: list[str],
```

Comment:
i don't really remember why the * was necessary, are we sure it isn't?

Reply:
I've just updated this - I don't think the unpacking was doing anything here. It is used in a couple of cool ways, but I don't think any were required here

https://stackabuse.com/unpacking-in-python-beyond-parallel-assignment/ (specifically sub-heading `Defining Functions With * and **`)

As I understand it, you started off with a list, `pv_files`. 
```[1, 2, 3, 4]```

That was being un-packed in the method call using `*pv_files`.
```summarise_association_results(1, 2, 3, 4, pv_all_filename_str)```

Then the method was also using `*` to pack all the arguments back into a list
```summarise_association_results(*pv_dfs, pv_all_filename_str)```

@illusional I think this makes sense and is safe to remove from both sides of the call & method sig, as we do actually want them in a list, so there's no need to unpack. Could you check the syntax?

Reply:
Yes, I believe this if fine because it's not a list of PythonResults, they're just lists of strings.